src/ContaCorrente.java

```diff
@@ -13,4 +13,12 @@ public void setLimite(double limite) {
             this.limite = limite;
         }
     }
+    public double calcularTaxas(){
+        if(getDono() instanceof PessoaFisica) {
```

Comment:
Matheus, essa implementação funciona, mas não é a ideal porque não segue as regras do polimorfismo. Polimorfismo permite tratar objetos de diferentes classes de maneira uniforme, sem precisar saber o tipo específico do objeto. Portanto, é necessário acessar diretamente o método calcularTaxas do dono.
Assim: return this.getDono().calcularTaxas().